.gitignore

```diff
@@ -41,3 +41,10 @@ dist/**/*
 
 # ignore yarn.lock
 yarn.lock
+example/feePayerPrivateKey
+example/privateKey
+example/privateKey2
+example/privateKey3
+example/publicKey
+example/publicKey2
+example/publicKey3
```

Comment:
Or name the key files `xxx.priv` `xxx.pub` and then ignore `example/*.priv` `example/*.pub`

Reply:
Good. I will update in web3klaytn.